### PR TITLE
Update config.guess and gmsl

### PR DESCRIPTION
**Pull request by Lord-Kamina**
We’re currently using two-(almost and-a-half) year old versions.
The new config.guess supports musl among other stuff.

Also, fixed their respective update scripts so they’ll work again.

### CODE REVIEW

**Comment by tonytheodore** [comment]
Were the update scripts failing when run from the top-level mxe dir (i.e `tools/update-*`)?

**Comment by Lord-Kamina** [comment]
Can't remember for sure if I tried to run them from top-level dir honestly.
Might have fucked that up.

**Comment by Lord-Kamina** [comment]
Reverted the changes to the update scripts; left only the actual updates to gmsl and config.guess.

File: docs/gmsl.html [hunk 1]
```diff
@@ -22,7 +22,7 @@ <h2>Using GMSL</h2>
 you have the right version of <span style="font-family: monospace;">gmsl</span>
 use the <span style="font-family: monospace;">gmsl_compatible</span>
 function (see
-below). The current version is <span style="font-family: monospace;">1 1 7</span>.<br>
+below). The current version is <span style="font-family: monospace;">1 1 6</span>.<br>
```
Comment (tonytheodore) [inline docs/gmsl.html]: 1 1 8

Comment (Lord-Kamina) [reply docs/gmsl.html]: Huh. Must have been a merge error when I edited the commit.

**Comment by tonytheodore** [comment]
Thanks!

**Pull request merged**